recipes/fiftyone-brain/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fiftyone-brain" %}
+{% set version = "0.16.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiftyone-brain-{{ version }}.tar.gz
+  sha256: a3f52bf5b7be1516f7b5202b434d8e15cf60e570b1b9d13652a0ad9610864b67
+
+build:
+  skip: false
+  # skip: true  # [py<37]
```

Comment:
```suggestion
```

Reply:
Looks like this can be noarch.

Reply:
You need to add the line to make it noarch. Please see https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
It seems that none of the packages use native extensions but depends on packages that use, like OpenCV and PyTorch.